src/components/Match/renderMatch.js

```diff
@@ -264,6 +264,17 @@ function renderMatch(m) {
     }
     return newPlayer;
   });
+
+  if (m.objectives && m.objectives.length > 0) {
+    m.objectives.map(obj => (
```

Comment:
Since you're not returning an array here, I prefer forEach, with a regular block statement (so you can use if directly instead of doing the && which I find harder to read